test/asyncio_tests/test_examples.py

```diff
@@ -974,13 +974,13 @@ async def run_transaction_with_retry(txn_coro, session):
                     break
                 except (ConnectionFailure, OperationFailure) as exc:
                     # If transient error, retry the whole transaction
-                    if exc.has_error_label("TransientTransactionError"):
-                        print("TransientTransactionError, retrying "
-                              "transaction ...")
-                        continue
-                    else:
+                    if not exc.has_error_label("TransientTransactionError"):
```

Comment:
Can you revert these changes as well? Same reason as before: it doesn't fit in 80 chars.

Once you revert, feel free to replace the "else: raise" with just "raise". That would clean it up a bit.